docs/yourbase-ruby.md

```diff
@@ -0,0 +1,73 @@
+# YourBase Ruby Acceleration
+
+Tests are important. For large monoliths, they're also a major source of drag on velocity.
+
+YourBase is a tool that traces your tests to determine which functions each test depends on. It can later use this information to determine which tests do not need to run because their code paths have not changed. These tests are skipped automatically.
+
+YourBase works with Ruby versions >= 2.3
+
+## Getting Started
+No configuration, setup, or babysitting required. To get started, you need the YourBase gem and a download token.  To request a token, please visit [YourBase.io](https://yourbase.io/download)
+
+Once you have a token, simply follow the steps below:
+```sh
+bundle add yourbase-rspec --git "https://${YOURBASE_DOWNLOAD_TOKEN?}:x-oauth-basic@github.com/yourbase/yourbase-rspec-skipper-engine.git" && bundle install
+```
+
+## First run
+After installing yourbase-rspec, if you are not using Rails you must add
+"require 'yourbase-rspec'" in your spec folder.
+
+```sh
+# Add require for non Rails projects
+echo "require 'yourbase-rspec'" >> spec/yourbase_spec.rb
+```
+
+Run your tests with the same command you typically use. You should see a rocket ship at the beginning the RSpec test section.
+
```

Comment:
We should have a section about Spring here, because it's the most common reason they won't see the rocketship. 

Right now, they need to run `spring stop` from the command line to turn it off, but it would be better if we could figure out if manually requiring `yourbase-rspec` at the top of the spec_helper.rb gets around the incompatibility, so that we could tell them to do that instead of having to stop Spring. Stopping Spring is definitely fine in the CI but is a little kludgier for local dev where Spring does actual useful things, including shorten the spin-up time for "loading the rails environment" for specs.

Reply:
Ok.  Have this in my notes but wanted to try it out first.  Will add it.

Reply:
I think for now it is sufficient to say that if you do not see the rocketship but ARE running Spring, that it should be turned off to accelerate the tests.